src/language/en-US.json

```
@@ -721,19 +735,7 @@
     "welcome_to_symbol": "Welcome to Symbol",
     "word": "Word",
     "x_seconds": "{seconds}s.",
-    "progress": "Download in progress.",
-    "contact_name": "Name",
-    "contact_address": "Address",
-    "contact_phone": "Phone",
-    "contact_email": "Email",
-    "contact_notes": "Notes",
-    "add_contact": "Add contact",
-    "backup_address_book": "Back up address book",
-    "form_label_new_contact_name": "New contact name",
-    "form_label_new_contact_address": "New contact address",
-    "delete_confirmation_message": "Are you sure do you want to delete?",
-    "tab_accounts_accounts": "My accounts",
-    "tab_accounts_addressbook": "My contacts",
-    "select_contact": "Select contact",
-    "contact_information": "Contact information"
+    "node": "Node",
+    "to_symbol": "to Symbol",
+    "program_description": "Based on the design of the Symbol blockchain chain, the open source Symbol digital asset system, it is convenient to access the symbol network, create accounts, manage assets and more operations..."
```

```suggestion
    "program_description": "The most convenient way to access the Symbol blockchain, the digital assets management system. Use it to create accounts, manage tokens and more!"
```

How does this string relate to `program_description[1,2,3]`? It will be harder to maintain 2 different versions!

---

this is not currently used in any page inside app and probably will be deleted

---

Haha, OK, good to see it removed then. It would be nice to have automated checks in place to ensure that all strings are in use so translator resources are not wasted.